cfn-resources/cloud-backup-restore-jobs/CHANGELOG.md

```diff
@@ -0,0 +1,75 @@
+# Changelog
+
+## (2023-10-05)
+
+**(BREAKING CHANGE) ADDED SUPPORT TO SERVERLESS INSTANCE**
+
+Originally, the resource partially supported serverless instances, but it required some schema changes to fully support
+the creation of restore jobs for serverless instances.
+
+- Deprecated fields:
+    - ClusterName
+- New fields:
+    - InstanceName: string
+    - InstanceType: string ["serverless", "cluster"]
+
+example:
+
+Original:
+``` yaml
+RestoreJob:
+Type: MongoDB::Atlas::CloudBackUpRestoreJobs
+Properties:
+ProjectId: !Ref 'ProjectSource'
+ClusterName: !Ref 'ClusterNameSource' <----- Old identifier (deprecated)
```

Comment:
This field has been completely removed right? In this case would modify `deprecated` with `removed`